scripts/local/launch.py

```diff
@@ -18,8 +18,13 @@ def _main() -> None:
     # Set up argparse.
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", required=True, type=str)
+    # Optionally override the branch specified in the CFG.
```

Comment:
i would suggest we don't use "CFG" here, this is a different config file than settings.py right?